### PR TITLE
cervical cancer cause model update

### PR DESCRIPTION
**Pull request by yongqx**

### CODE REVIEW

File: docs/source/gbd2017_models/causes/neoplasms/cervical_cancer/cervical_cancer_cause_model.rst [hunk 1]
```diff
@@ -10,10 +10,14 @@ Cervical Cancer
...
-.. todo::
-
-   Add definition of cervical cancer. In particular, find data about global 
-   prevalence and disease fatal and non fatal description.
+Cervical cancer is a female-specific cancer. It is prevalent globally, ranked 
+as the fourth-most common cancer in women. In 2018, about 106,430 new cervical 
+cancer cases are diagnosed in China and about 47,739 cervical cancer deaths occur 
+annually in China. For women at 50-54 years of age, the incidence of cervical 
+cancer reaches its maximum value, 30 cases per 100,000 person-years. The deaths 
+due to cervical cancer are increase over age, has the highest value 29 per
```
Comment (aflaxman) [inline docs/source/gbd2017_models/causes/neoplasms/cervical_cancer/cervical_cancer_cause_model.rst:18]: ~~are~~

Comment (aflaxman) [reply docs/source/gbd2017_models/causes/neoplasms/cervical_cancer/cervical_cancer_cause_model.rst:18]: has -> have

Comment (aflaxman) [reply docs/source/gbd2017_models/causes/neoplasms/cervical_cancer/cervical_cancer_cause_model.rst:18]: add "and" before have

File: docs/source/gbd2017_models/causes/neoplasms/cervical_cancer/cervical_cancer_cause_model.rst [hunk 2]
```diff
@@ -160,21 +168,33 @@ State and Transition Data Tables
...
-   * - S
+     - No EMR for susceptible state
+   * - Susceptible
      - disabilty weights
      - 0
-     - No disability weights for S state
+     - No disability weights for susceptible state
+   * - hrHPV-infected
+     - prevalence
+     - 19.0% (95%CI, 17.1-20.9)
```
Comment (aflaxman) [inline docs/source/gbd2017_models/causes/neoplasms/cervical_cancer/cervical_cancer_cause_model.rst:185]: We will need to include an age-pattern for this, and if I have understood correctly, there is not an evidence base of incidence data to draw on.  Let's include information on how we proceed in this document.

**Review by aflaxman — APPROVED** [review]
This will make a good test case for the "concept model implementation" idea I have been thinking about.  I'll follow up about that on Slack.

**Review by plinds — APPROVED** [review]

**Pull request merged**